python/common/org/python/types/Str.java

```diff
@@ -1764,8 +1764,42 @@ public static String _title(String input) {
     public org.python.Object upper() {
         return new org.python.types.Str(this.value.toUpperCase());
     }
-}
 
+    @org.python.Method(
+            __doc__ = "S.zfill() -> str\n" +
+                    "\n" +
+                    "Return a copy of the string left filled with ASCII '0' \n" +
+                    "digits to make a string of length width.\n",
+            args = {"width"}
+    )
+    public org.python.Object zfill(org.python.Object width) {
+
+        if (width == null) {
+            throw new org.python.exceptions.TypeError("zfill() takes exactly 1 argument (0 given)");
```

Comment:
you can remove this error handling too, `width` will never be null with the declaration of arguments as `args` (it might be `None` though, which would still be a org.python.Object)